### ISSUE TITLE
Bug: React 16.0.0 breaks Transition

### ISSUE THREAD

**Issue by taylorlapeyre** [reactjs/react-transition-group#197]
![screen shot 2017-09-26 at 12 11 49 pm](https://user-images.githubusercontent.com/1610503/30879342-0052f432-a2b4-11e7-8848-eb19f8f1db0b.png)

Seems like this library needs to accommodate React's new updating system?

**Comment by jquense** [comment]
The api there hasn't changed in v16 of React so I'm not sure what could break. I'm not seeing this error

**Comment by taylorlapeyre** [comment]
@jquense I don't remember seeing any API changes listed in the docs either, but something certainly seems different!

![screen shot 2017-09-26 at 12 35 59 pm](https://user-images.githubusercontent.com/1610503/30880382-4d71f3b4-a2b7-11e7-92ac-2440c5c1f71d.png)

**Comment by duro** [comment]
I am seeing this as well. Looks like `enqueueCallback` is gone.

**Comment by jquense** [comment]
the issue is that no code here is calling that. sounds like maybe you forgot to upgrade react-dom as well?

**Comment by duro** [comment]
@jquense and @taylorlapeyre I saw this issue described in #151 and based on what I ready there it looks like the issue has to do with having multiple version of react in your deps.

I am using yarn, so I went ahead an manually edited the yarn.lock to remove the react 15 bundle that was still being installed. This cause some of my packages to to complain about having the wrong peer-dependencies, but all my tests are passing, and app is working well.

TLDR; This is not an issue with react-transition-group

**Comment by taylorlapeyre** [comment]
Thanks @duro !

**Comment by cezarderevlean** [comment]
Multiple versions of react might happen when there's another package depending on it.
Material UI v1 is such a package.
